Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -326,6 +326,10 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
                 response.signature = toArrayBuffer(credential.get().signature);
                 response.userHandle = toArrayBuffer(credential.get().userID);
             }
+            if (m_cancelHandler) {
```

Comment:
I think you can do the fancy `if (auto cancelHandler = WTFMove(m_cancelHandler))` here.